### PR TITLE
Olwa/117

### PR DESCRIPTION
**Pull request by owalk**
Entities now load from the maps created by Tiled map editor.

### CODE REVIEW

**Comment by owalk** [comment]
The surface key color logic is broken :(

I've made a new [issue](https://github.com/Sean-Nishi/Reluctant-Hero/issues/159) to fix this later.

File: game/levels/level.py [hunk 1]
```diff
@@ -52,6 +57,9 @@ def __init__(self, level_data, surface):
         self.friendly_sprites = pygame.sprite.Group()
         self.attack_sprites = pygame.sprite.Group()
 
+        # add stuff to their sprite groups
```
Comment (Sean-Nishi) [inline game/levels/level.py:60]: Remove 60-62, out of commit scope.

File: game/levels/level.py [hunk 2]
```diff
@@ -63,12 +71,52 @@ def __init__(self, level_data, surface):
...
 
+    def create_entities_from_layout(self, layout):
+        """Initialize the entities on a layout.
+
+        Parameters
+        ----------
+        layout: array of values each representing an individual entity
+        """
+
+        sprite_group = pygame.sprite.Group()
```
Comment (Sean-Nishi) [inline game/levels/level.py:92]: remove this line4

File: game/levels/level.py [hunk 3]
```diff
@@ -63,12 +71,52 @@ def __init__(self, level_data, surface):
...
+        x = 0
```
Comment (Sean-Nishi) [inline game/levels/level.py:93]: doc that this is the default pos

**Review by Sean-Nishi — CHANGES_REQUESTED** [review]
Looks good except for out of scope comments.

**Review by Sean-Nishi — APPROVED** [review]

**Pull request merged**